config/301-awss3target.yaml

```diff
@@ -88,7 +88,7 @@ spec:
                 type: string
                 description: ARN of the S3 bucket that will receive events. The expected format is documented at
                   https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazons3.html
-                pattern: ^arn:aws(-cn|-us-gov)?:s3:[a-z]{2}(-gov)?-[a-z]+-\d:\d{12}:.+$
+                pattern: ^arn:aws(-cn|-us-gov)?:s3:::.+$
```

Comment:
nit: The actual naming requirement is slightly more restrictive than `.+`.
Being strict is a good way to prevent loops of errors early if a user passes input that's clearly not gonna work.

https://github.com/triggermesh/triggermesh/blob/3ce81f86e06a3d407c8b52029b133547c30b3234/config/300-awss3source.yaml#L72